docs/UserGuide.md

```diff
@@ -3,34 +3,42 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
-
-* Table of Contents
-{:toc}
+Weeblingo is a desktop app for managing flashcards, **optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). With a nice and friendly interface, users can learn Japanese at a comfortable pace with this application.
+
+* [Quick Start](#quick-start)
+* [Features](#features)
+  * [Starting a session: `start`](#starting-a-session-start)
+  * [Checking flashcard answers: `check`](#checking-flashcard-answers-check)
+  * [Going to next flashcard: `next`](#going-to-next-flashcard-next)
+  * [Listing out all flashcards: `list`](#listing-out-all-flashcards-list)
+  * [Ending the session: `end`](#ending-the-session-end)
+  * [Exiting the application: `exit`](#exiting-the-application-exit)
+  * [Asking for help: `help`](#asking-for-help-help)
+    
+* [FAQ](#faq)
+* [Command Summary](#command-summary) 
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick start
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `weeblingo.jar` from [here](https/linkhere-tbc).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy the file to the folder you want to use as the _home folder_ for your WeebLingo application.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. <br>
```

Comment:
The GUI similar to the **picture** below

Reply:
Fixed!